apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -364,11 +364,19 @@ function replaceInvalidControlValues(
 ): Record<string, unknown> {
   const fixedValues = _.cloneDeep(normalizedControlValues);
 
-  errors.forEach((error) => {
+  for (const error of errors) {
+    /*
+     *  we allow additional properties in control values compare to output
+     *  such as skip and disableOutputSanitization
+     */
+    if (error.keyword === 'additionalProperties') {
+      continue;
+    }
+
```

Comment:
at the moment the preview will fail because the code bellow does not know how to handle additional params error.